sdk/v2/builder.go

```diff
@@ -26,10 +27,17 @@ type IntegrationBuilder interface {
 	// Writer sets the output stream where the integration resulting payload will be written to.
 	// By default, the standard output (os.Stdout).
 	Writer(io.Writer) IntegrationBuilder
+	// Cache sets the Cache implementation that will be used to persist data between executions of the same integration.
+	// By default, it will be a Disk-backed cache named stored in the file returned by the
+	// cache.DefaultPath(integrationName) function.
+	Cache(cache.Cache) IntegrationBuilder
+	// NoCache disables the cache for this integration.
+	NoCache() IntegrationBuilder
```

Comment:
Not sure about this.

Integrations do need a persistent cache, I guess we should create a one per integration and hide it to the user.

Why a user would like to set it's own cache? or remove the current one?

Reply:
I think that integrations without deltas don't need a cache.

Also, they could prefer using other caches, like redis, etcd, etc....